examples/routing_service/routing_service_with_security/governance.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/6.0.0/dds_security_governance.xsd">
+  <domain_access_rules>
+    <domain_rule>
+      <domains>
+        <id_range>
+		  <!-- This Governance will apply to Domains 1 and 2 -->
```

Comment:
Make sure we have a consistent style for comments (including multi-line comments). Also, fix the indentation.

Reply:
Done.